### PR TITLE
README: add contribution guidelines

### PR DESCRIPTION
**Pull request by mythi**
[skip-ci]

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -110,3 +110,34 @@ Installing the Images
...
+.. _here: https://lists.yoctoproject.org/listinfo/intel-iot-refkit
+
+Bug Tracking
+============
+
+The project uses the `Yocto Project Bugzilla`_ for feature/enhancement and bug tracking.
+
+If you've found a bug or wish to file an enhancement request, go to Bugzilla
+and submit the tickets against Yocto Project Subprojects | IoT Reference OS Kit
+component.
```
Comment (pohly) [inline README.rst]: I don't see such a "IoT Reference OS Kit" under "Yocto Project Subprojects".

Comment (pohly) [inline README.rst]: Never mind, found it now. I was just searching for the wrong term (refkit...)

Comment (gvancuts) [inline README.rst]: We could also use this URL to make it easier for users to file a bug or feature request for the IoT Reference OS Kit: https://bugzilla.yoctoproject.org/enter_bug.cgi?product=IoT%20Reference%20OS%20Kit

**Comment by mythi** [comment]
v2: addressed the review feedback, rebased and resolved conflicts.

**Pull request merged**